release.sh

```diff
@@ -11,7 +11,7 @@ patch_re=\#Patch
 
 echo ${log_messages}
 
-args="-n --npm.access=public --github.release --no-git.requireCleanWorkingDir --github.assets=build/database.sqlite"
+args="-n --npm.access=public --github.release --no-git.requireCleanWorkingDir --no-git.requireUpstream --github.assets=build/database.sqlite"
```

Comment:
Maybe also add a CI flag similar to how it was done in GU

https://github.com/shabados/gurmukhi-utils/pull/184

Reply:
This `--npm.skipChecks`?

Reply:
Yes